src/SFA.DAS.EAS.Web/Orchestrators/Mappers/ApprenticeshipMapper.cs

```diff
@@ -118,6 +118,11 @@ public ApprenticeshipViewModel MapToApprenticeshipViewModel(Apprenticeship appre
                                     apprenticeship.StartDate.HasValue &&
                                     _academicYearValidator.Validate(apprenticeship.StartDate.Value) == AcademicYearValidationResult.NotWithinFundingPeriod;
 
+            var isApprovedTransferAndNoSuccessfulIlrSubmission =
+                commitment.TransferSender != null
+                && commitment.TransferSender.TransferApprovalStatus == TransferApprovalStatus.Approved
```

Comment:
does it make sense to write this as commitment.TransferSender?.TransferApproval == ....

Reply:
Yes, applied simplification.